mava/wrappers/jumanji.py

```diff
@@ -117,3 +124,96 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
 
         # Aggregate the list of individual rewards and use a single team_reward.
         return self.aggregate_rewards(timestep, modified_observation)
+
+
+class ConnectorWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the MA Connector environment.
+
+    Do not use the AgentID wrapper with this env, it has implicit agent IDs.
+    """
+
+    def __init__(self, env: MaConnector, has_global_state: bool = False):
+        super().__init__(env)
+        self.has_global_state = has_global_state
```

Comment:
Will MaConnector have a global state?

Reply:
Ye, it's just the obs of agent 0

Reply:
Yes. The global state is the agent observation without the per_agent parts. So just a full view of the grid.